lib/iris/tests/test_grib_save.py

```diff
@@ -34,6 +34,7 @@
 
 if tests.GRIB_AVAILABLE:
     import gribapi
+    from iris_grib._load_convert import _MDI as MDI
```

Comment:
As a private API, we obviously we shouldn't be relying on this, but happy enough for now.